src/Persistence.php

```diff
@@ -18,6 +20,21 @@ class Persistence
     use \atk4\core\NameTrait;
     use \atk4\core\DiContainerTrait;
 
+    // backward compatibility - will be removed in dec-2020 -->
```

Comment:
this is new, why it should be removed later?

Reply:
We have hook constants in Persistence forwarded to hook constants in AbstractQuery.

We can keep it like it is without deprecation, I am for it but up for discussion.